src/content/en/fundamentals/design-and-ui/input/touch/_code/touch-demo-1.html

```diff
@@ -1,17 +1,7 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <meta charset="utf-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1">
-    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
-    <link rel="stylesheet" href="https://code.getmdl.io/1.2.1/material.indigo-pink.min.css">
-    <script defer src="https://code.getmdl.io/1.2.1/material.min.js"></script>
-    <style>
-      body {
-        margin: 2em;
-      }
-    </style>
+    <!-- // [TEMPLATE header_full] -->
```

Comment:
The whole head needs to be included, the build system doesn't automatically insert or handle  `<!-- // [TEMPLATE header_full] -->`